rust/parser/typeql.pest

```diff
@@ -5,90 +5,61 @@
  */
 
 eof_query = { SOI ~ query ~ EOI }
-eof_queries = { SOI ~ query+ ~ EOI }
-eof_pattern = { SOI ~ pattern ~ EOI }
-eof_patterns = { SOI ~ patterns ~ EOI }
-eof_definables = { SOI ~ definables ~ EOI }
-eof_statement = { SOI ~ statement ~ EOI }
 eof_label = { SOI ~ label ~ EOI }
 eof_definition_function = { SOI ~ definition_function ~ EOI }
+eof_definition_struct = { SOI ~ definition_struct ~ EOI }
 
 // TYPEQL QUERY LANGUAGE =======================================================
 
 query = { query_schema | query_data }
 
-query_schema = { query_define | query_undefine }
-query_define = { DEFINE ~ definables }
-query_undefine = { UNDEFINE ~ definables }
-
-query_data = { query_stage+ }
-query_stage = { stage_match
-              | stage_insert
-              | stage_delete
-              | stage_put
-              | stage_fetch
-              | stage_reduce
-              | stage_modifier
-              }
+// DATA QUERIES ================================================================
```

Comment:
Let's call all this `Query Pipelines`, which we can leave as asymmetric with `query_schema | query_pipeline` atm